pset2/caesar/caesar.c

```diff
@@ -25,10 +36,11 @@ int main(int argc, string args[])
             continue;
         }
 
-        int ascii_offset = isupper(plaintext[i]) ? 65 : 97;
+        // Uppercase letter's index has a different offset than the lowercase counterpart in ascii table
+        int ascii_offset = isupper(plaintext[i]) ? UPPERCASE_LETTERS_START : LOWERCASE_LETTERS_START;
 
```

Comment:
//Does the "UPPERCASE_LETTERS_START : LOWERCASE_LETTERS_START" define the range between the ascii set ? and why do you have to subtract it from the plaintext?